allennlp/models/archival.py

```diff
@@ -128,7 +128,11 @@ def archive_model(
 
 
 def load_archive(
-    archive_file: str, cuda_device: int = -1, overrides: str = "", weights_file: str = None
+    archive_file: str,
+    weights_file: str = None,
+    cuda_device: int = -1,
+    opt_level: str = None,
```

Comment:
I was thinking about whether we should have some sentinel (`DEFAULT = object()`, defined in `common.util` or something), so that you could actually specify `None`, but after thinking about it a bit, I'm not sure it's worth it at this point.  If someone runs into a case where they want to specify `None` here and have it mean `None`, then we can add the complexity to handle it.  Not sure it's necessary at this point.